Introduces separate runtime provider schema (#13488)
mistercrunch/Airflow: docs/apache-airflow-providers/index.rst

* Add the ``apache_airflow_provider`` entry point in the ``setup.cfg`` - this tells airflow where to get
  the required provider metadata
* Create the function that you refer to in the first step as part of your package: this functions returns a
  dictionary that contains all meta-data about your provider package
* note that the dictionary should be compliant with ``airflow/provider_info.schema.json`` JSON-schema

>   specification, and the community-managed providers have more fields there that are used to build

Reviewer: ```suggestion
  specification and the community-managed providers have more fields than that is used to build
```

I am assuming this is what you meant but feel free to ignore if not
